volue_insight_timeseries/util.py

```diff
@@ -56,6 +57,7 @@
     '15T': 'MIN15',
     '5T': 'MIN5',
     'T': 'MIN',
+    'D': 'D',
```

Comment:
this line won't be needed since we update _PANDAS_FREQ_TABLE with values from _TS_FREQ_TABLE on line 62,